pkg/ctl/create/iamidentitymapping.go

```diff
@@ -14,45 +11,40 @@ import (
 	"github.com/weaveworks/eksctl/pkg/eks"
 )
 
-func createIAMIdentityMappingCmd(g *cmdutils.Grouping) *cobra.Command {
-	p := &api.ProviderConfig{}
+func createIAMIdentityMappingCmd(rc *cmdutils.ResourceCmd) {
```

Comment:
> it would be more idiomatic if it was the receiver of these methods

yes, but that means all those have to be in the same package...

Reply:
> Since I don't normally expect a parameter of a function to be mutated by it.

It's unfortunate, but we already have too many places where `*api.ClusterConfig` is passed in and mutated, which is kind of by design. We can discuss how that can be fix, but I'm not entirely sure if it the most important thing at the moment. (Only if there was a way in Go to prevent mutation!)